README.md

```diff
@@ -1,77 +1,19 @@
 # scikit-learn-knn-regression
 
-This package is in active development.
+> [!WARNING]  
+> This package is in active development!
 
-## Developer Guide
+`scikit-learn-knn-regression` (a.k.a. `sknnr`) is a package for running k-nearest neighbor imputation methods, including GNN (Ohmann & Gregory, 2002), MSN (Moeur & Stage 1995), and RFNN (Crookston & Finley, 2008), using estimators that are fully compatible with [`scikit-learn`](https://scikit-learn.org/stable/).
 
-### Setup
 
-This project uses [hatch](https://hatch.pypa.io/latest/) to manage the development environment and build and publish releases. Make sure `hatch` is [installed](https://hatch.pypa.io/latest/install/) first:
+## Acknowledgements
 
-```bash
-$ pip install hatch
-```
+`sknnr` was inspired by the [yaImpute](https://cran.r-project.org/web/packages/yaImpute/index.html) package for R (Crookston & Finley 2008). Thanks to Andrew Hudak for the inclusion of the [Moscow Mountain / St. Joes dataset](api/datasets/moscow_stjoes.md) (Hudak 2010), and Tom DeMeo for the inclusion of the [SWO Ecoplot dataset](api/datasets/swo_ecoplot.md) (Atzet et al., 1996). Development of this package was funded in part by an appointment to the United States Forest Service (USFS) Research Participation Program administered by the Oak Ridge Institute for Science and Education (ORISE) through an interagency agreement between the U.S. Department of Energy (DOE) and the U.S. Department of Agriculture (USDA).
```

Comment:
I wanted to get these down so I didn't forget to add this section, but you have a much better idea of who needs thanking for what than I do, so please edit as needed! I added my ORISE funding info, but I imagine we want to mention LEMMA OSU here or elsewhere? We could probably add a logo or some branding throughout the docs as well.

Reply:
For now, I haven't done anything to `README.md` given what you found about syncing content between doc pages and package documentation.  

In `index.md`, however, I've created a new section called `History and Inspiration` that details the tie to `yaImpute` and then added my grants associated with this project into the acknowledgements section.

Reply:
Perfect, those changes all look great to me!